From 729d799b844630f8f6eb34dcbaf3ae7fdfd3bf7f Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Thu, 25 Jan 2018 14:08:54 -0500
Subject: [PATCH] RELENG-612 prefer (git+)https urls Reviewed by: Trent Mick
 <trentm@gmail.com> Approved by: Trent Mick <trentm@gmail.com>

---
 .gitmodules  | 6 +++---
 package.json | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index aca6e0c..eb36562 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,12 +1,12 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/jsdoc3"]
 	path = deps/jsdoc3
-	url = git@github.com:jsdoc3/jsdoc.git
+	url = https://github.com/jsdoc3/jsdoc.git
 	branch = remotes/origin/releases/3.1
 [submodule "deps/json"]
 	path = deps/json
diff --git a/package.json b/package.json
index b68a387..c4f3cba 100644
--- a/package.json
+++ b/package.json
@@ -48,8 +48,8 @@
     },
     "devDependencies": {
         "byline": "4.1.1",
-        "nodeunit": "git://github.com/yunong/nodeunit.git#4b1bf5e52941b72cf619a246f87b261da95a1231",
-        "node-manatee": "git://github.com/joyent/node-manatee#master",
+        "nodeunit": "git+https://github.com/yunong/nodeunit.git#4b1bf5e52941b72cf619a246f87b261da95a1231",
+        "node-manatee": "git+https://github.com/joyent/node-manatee#master",
         "tap": "0.4.8"
     },
     "scripts": {
-- 
2.21.0

